lib/matplotlib/tests/test_colors.py

```diff
@@ -1175,3 +1175,10 @@ def test_get_under_over_bad():
     assert_array_equal(cmap.get_under(), cmap(-np.inf))
     assert_array_equal(cmap.get_over(), cmap(np.inf))
     assert_array_equal(cmap.get_bad(), cmap(np.nan))
+
+
+def test_2d_to_rgba():
+    color = np.array([0.1, 0.2, 0.3])
+    rgba_1d = mcolors.to_rgba(color.reshape((-1)))
```

Comment:
```suggestion
    rgba_1d = mcolors.to_rgba(color.reshape(-1))
```

Reply:
I have changed all occurrences of .rebase((-1)) and performed a rebase, please verify.